titus-supplementary-component/task-relocation/src/main/java/com/netflix/titus/supplementary/relocation/workflow/step/TaskEvictionStep.java

```diff
@@ -90,9 +90,20 @@ public TaskEvictionStep(EvictionServiceClient evictionServiceClient,
                 .collect(Collectors.toMap(
                         TaskRelocationPlan::getTaskId,
                         p -> {
-                            String message = String.format("%s: reasonCode=%s, plannedRelocationTime=%s",
-                                    p.getReasonMessage(), p.getReason(), DateTimeExt.toUtcDateTimeString(p.getRelocationTime())
-                            );
+                            String message;
+                            switch (p.getReason()) {
```

Comment:
Do we need to handle `null` reason value?

Reply:
Good catch. I will make sure in the builder that this value is always set to something.